tests/test_io.py

```diff
@@ -64,6 +68,8 @@ class TestExtractors(TestCase):
     @parameterized.expand(imaging_extractor_list, name_func=custom_name_func)
     def test_imaging_extractors(self, extractor_class, extractor_kwargs):
         extractor = extractor_class(**extractor_kwargs)
+        assert_get_frames_indexing_with_single_channel(imaging_extractor=extractor)
+
```

Comment:
Could you make this a separate test function that still uses the same `parameterized.expand` over the `imaging_extractor_list`? That way `test_imaging_extractors` stays as the simple read/write roundtrip and nothing more

Reply:
I think that I am more concerned with you that our tests will become slower and slower and this moves us toward that direction. On the other hand, it is better to keep the tests isolated and I think this is a stronger concern here. I will change this.

Reply:
Test runtime should be unaffected either way, the only common overhead is `__init__` of each extractor, which I certainly hope is not a heavy operation.

Reply:
Yes, my thinking is that it should not be, depends on how we are loading the videos. But overall I agree, we keep the test isolated now and then we should reduce **if** they become too slow.

Reply:
Should be done, I also removed the testing of the video shape as this is test should cover the same. Moreover, I changed the name of the assert function to something less specific as I think if we decide to move on with only supporting slide we can just change the the current test to account for that.

Reply:
@CodyCBakerPhD 
Thinking further about this, I think that the `__-init__` of each of the extractors is going to be the heaviest operation together with `get_video`. As all the information and in some cases all the data is loaded at the level.